### PR TITLE
Get docs compiling

### PR DESCRIPTION
**Pull request by cmhyett**
## Checklist

- [ ] Appropriate tests were added
- [ ] Any code changes were done in a way that does not break public API
- [ ] All documentation related to code changes were updated
- [ ] The new code follows the
  [contributor guidelines](https://github.com/SciML/.github/blob/master/CONTRIBUTING.md), in particular the [SciML Style Guide](https://github.com/SciML/SciMLStyle) and
  [COLPRAC](https://github.com/SciML/COLPRAC).
- [ ] Any new documentation only uses public API
  
## Additional context

This is definitely a WIP, just pathfinding to find the offending portions in the docs. Nowhere ready for a review.

### CODE REVIEW

File: test/pde_systems/MOL_1D_Integration.jl [hunk 1]
```diff
@@ -145,16 +145,16 @@ end
 
     disc = MOLFiniteDifference([x => 120], t)
 
-    prob = discretize(pde_system, disc)
+    @test_throws ArgumentError (prob = discretize(pde_system, disc))
```
Comment (ChrisRackauckas) [inline test/pde_systems/MOL_1D_Integration.jl:148]: I don't think this is supposed to throw an error?

Comment (cmhyett) [reply test/pde_systems/MOL_1D_Integration.jl:148]: Moving this conversation to the test PR.

File: docs/make.jl [hunk 2]
```diff
@@ -12,7 +12,7 @@ makedocs(sitename = "MethodOfLines.jl",
          authors = "Chris Rackauckas, Alex Jones et al.",
          clean = true, doctest = false, linkcheck = true,
          modules = [MethodOfLines],
-         warnonly = [:docs_block, :missing_docs, :cross_references],
+         warnonly = true,#[:docs_block, :missing_docs, :cross_references],
```
Comment (ChrisRackauckas) [inline docs/make.jl:15]: You can instead turn on debug mode. https://github.com/SciML/DiffEqFlux.jl/blob/master/.buildkite/pipeline.yml#L39C20-L40C37

Comment (cmhyett) [reply docs/make.jl]: This change was reverted, and the suggestion of setting `JULIA_DEBUG="Documenter"` was taken.

**Comment by cmhyett** [comment]
@xtalax @ChrisRackauckas is there a timeout on how long the docs action can run? I'm now able to build the docs locally, but the action keeps getting cancelled.

File: .github/workflows/Documentation.yml [hunk 3]
```diff
@@ -14,11 +14,12 @@ jobs:
       - uses: actions/checkout@v4
       - uses: julia-actions/setup-julia@latest
         with:
-          version: '1'
+          version: '1.10'
```
Comment (ChrisRackauckas) [inline .github/workflows/Documentation.yml:17]:
```suggestion
          version: '1'
```

we should keep it on latest

**Comment by ChrisRackauckas** [comment]
@thazhemadam can we get this one to buildkite too?

**Comment by thazhemadam** [comment]
> @thazhemadam can we get this one to buildkite too?

@ChrisRackauckas does this need to be run on GPU machines? Or will the normal CPU runners suffice?

**Comment by ChrisRackauckas** [comment]
CPU runners are fine

**Comment by thazhemadam** [comment]
> @thazhemadam can we get this one to buildkite too?

This has been implemented in https://github.com/SciML/MethodOfLines.jl/pull/362.

**Comment by cmhyett** [comment]
Going to close this PR as I believe it's superseded by #362

**Comment by ChrisRackauckas** [comment]
No it's complementary.

**Pull request merged**